website/docs/examples/people_detection/people_detection.md

```diff
@@ -0,0 +1,94 @@
+---
+id: people-detection
+title: People Detection
+---
+
+
+### [Link to `people_detection/main.py`](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/people_detection/main.py)
+
+This example shows how to use the `farm-ng-amiga` library to detect people in a video stream.
+
+It also shows how to implement a service and client via grpc.
+
+The requirements to run this example are to have a [**farm-ng brain**](/docs/brain/) running Oak cameras and that your PC is on the same local network as the brain.
+
+For testing you can use your webcam as a replacement, which we will go over later in this tutorial.
+
+### 1. Install the [farm-ng Brain ADK package](/docs/brain/brain-install)
+
+### 2. Setup
+:::tip
+
+It is recommended to also install these dependencies and run the example in the brain ADK virtual environment.
+
+:::
+
+Create first a virtual environment
+```bash
+python3 -m venv venv
+source venv/bin/activate
+```
+
+```bash
+#assuming you have the farm-ng-amiga repository.
+cd farm-ng-amiga/py/examples/people_detection
+```
+### 3. Install Dependencies
+```bash
+pip3 install -r requirements.txt
+```
+### 3. Download the model data
+In this example we use MobileNet SSD from tensorflow to be implemented in opencv.
+
+Download the model weights and architecture:
+
+```bash
+mkdir models
+```
+```bash
+wget https://github.com/rdeepc/ExploreOpencvDnn/raw/master/models/frozen_inference_graph.pb -O models/frozen_inference_graph.pb
+```
+```bash
+wget https://github.com/rdeepc/ExploreOpencvDnn/raw/master/models/ssd_mobilenet_v2_coco_2018_03_29.pbtxt -O models/ssd_mobilenet_v2_coco_2018_03_29.pbtxt
+```
+### 4. Execute the Python script
```

Comment:
i would have two separated steps sections to run the Service and Client
4. Run the Service
you can add a note that this could run in your computer; or in the amiga
5. run the Client
same a above; maybe you can mention in between about specifying correctly the ip addresses

Also worth to explain the workflow like the server is a process that receives a decoded image, compute an AI model and return results. So, if you have the service in your machine, and the client consume from the robot, there's the transmission overhead of grpc from robot-camera to you laptop (service).